src/mc_control/ControllerClient.cpp

```diff
@@ -699,7 +699,15 @@ void ControllerClient::handle_form(const ElementId & id, const mc_rtc::Configura
         form_array_input(id, name, required, el[3], el[4], el.size() > 5 ? el[5] : true);
         break;
       case Elements::ComboInput:
-        form_combo_input(id, name, required, el[3], el[4]);
+        if(el.size() == 5)
+        {
+          // For backward compatibility
+          form_combo_input(id, name, required, el[3], el[4], -1);
+        }
+        else
+        {
+          form_combo_input(id, name, required, el[3], el[4], el[5]);
+        }
```

Comment:
```cpp
form_combo_input(id, name, required, el[3], el[4], el.size() > 5 ? el[5] : -1);
```

Reply:
fixed in https://github.com/jrl-umi3218/mc_rtc/pull/178/commits/2f5054bdc0a6757e6ab6f0a6586fa435ad37cbaf